package.json

```diff
@@ -22,7 +22,7 @@
     "colors-tmpl": "~0.1.0",
     "core-util-is": "~1.0.1",
     "cpr": "~0.3.2",
-    "getos": "~1.0.1",
+    "getos": "^2.3.0",
```

Comment:
except .. everything else is `~` so this is wrong

will fix in a separate commit and release as a patch bump